packages/app/src/domain/international/variants-stacked-area-tile/logic/use-unreliable-data-annotations.tsx

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import { isDefined } from 'ts-is-present';
+import { TimespanAnnotationConfig } from '~/components/time-series-chart/logic';
+import { VariantChartValue } from '~/static-props/variants/get-variant-chart-data';
+
+export function useUnreliableDataAnnotations(
+  values: VariantChartValue[],
+  label: string
+) {
+  return useMemo(
+    () =>
+      values
+        .filter((x) => isDefined(x.is_reliable) && x.is_reliable === false)
```

Comment:
Does TS accept this? It's practically the same:
```suggestion
        .filter((x) => x.is_reliable === false)
```

Reply:
True, but in theory, data could be passed in without an `is_reliable` property (this type is shared between national and into), and if the is_reliable is undefined, it is assumed to be reliable.

Reply:
Yeah but:
```tsx
({ is_reliable: false     }.is_reliable === false) === true
({ is_reliable: undefined }.is_reliable === false) === false
({                        }.is_reliable === false) === false

Reply:
haha, oh wait a sec, I had my boolean logic backwards, you're absolutely right haha